product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx

```diff
@@ -27,29 +27,79 @@ helm upgrade --dependency-update \
   --create-namespace \
   edb/edb-postgres-distributed-for-kubernetes \
   --set edb-postgres-for-kubernetes.enabled=true \
-  --set image.repository=docker.enterprisedb.com/<repository_name>/pg4k-pgd \ 
-  --set edb-postgres-for-kubernetes.image.repository=docker.enterprisedb.com/<repository_name>/edb-postgres-for-kubernetes \
-  --set config.data.PGD_IMAGE_NAME=docker.enterprisedb.com/<repository_name>/<operand_image_name> \
-  --set config.data.PGD_PROXY_IMAGE_NAME=docker.enterprisedb.com/<repository_name>/<proxy_image_name>
-  --set image.imageCredentials.username=<repository_name> \
-  --set image.imageCredentials.password=<your_repository_token>
+  --set image.repository=docker.enterprisedb.com/<REPOSITORY_NAME>/pg4k-pgd \ 
+  --set edb-postgres-for-kubernetes.image.repository=docker.enterprisedb.com/<REPOSITORY_NAME>/edb-postgres-for-kubernetes \
+  --set config.data.PGD_IMAGE_NAME=docker.enterprisedb.com/<REPOSITORY_NAME>/<OPERAND_NAME> \
+  --set config.data.PGD_PROXY_IMAGE_NAME=docker.enterprisedb.com/<REPOSITORY_NAME>/<PGD-PROXY>
+  --set image.imageCredentials.username=<REPOSITORY_NAME> \
+  --set image.imageCredentials.password=<EDB_SUBSCRIPTION_TOKEN>
 ```
 
 After assembling the command with the required images, 
-see [Installation](/postgres_distributed_for_kubernetes/latest/installation_upgrade/) for instructions on how to add the repository, 
+see [Installation](../installation_upgrade) for instructions on how to add the repository, 
 deploy the images, and create a certificate issuer. 
 
-## Example 
+For more information about how to assemble your command, see the examples: 
 
-The example command: 
+## Examples 
 
-* Pulls images from the `k8s_enterprise_pgd` repository. 
-* Uses EBD Postgres Advanced Server 15.6.2 as the Postgres option. 
-* Uses PGD 5.4.1 as the Postgres Distributed version.
-* Uses 5.4.0 as the PGD Proxy version.
+These example commands: 
+
+* Pull images from the `k8s_enterprise_pgd` repository. 
+* Use EBD Postgres Advanced Server 15.6.2 as the Postgres option. 
+* Use PGD 5.4.1 as the Postgres Distributed version.
+* Use 5.4.0 as the PGD Proxy version.
+
+### Example 1: Setting the environment variables before you run the deployment command
```

Comment:
as before, if the Examples section was a subsection, then this should be an h4...